devices/src/legacy/rtc_pl031.rs

```diff
@@ -8,16 +8,17 @@
 //! a real-time clock input.
 //!
 
+extern crate fc_util;
+
 use std::fmt;
-use std::ops::Add;
 use std::{io, result};
 
 use byteorder::{ByteOrder, LittleEndian};
 
 use crate::BusDevice;
 use logger::{Metric, METRICS};
+use std::time::Instant;
```

Comment:
Please group imports together:
* first, imports from `std` (i.e. this line should move under the other `use std::`s)
* then, imports from other crates from crates.io
* last, imports from other crates in Firecracker